FiMAdminApi.Clients/OrangeAllianceDataClient.cs

```diff
@@ -120,6 +120,50 @@ public async Task<bool> UpdateEventStream(
         return true;
     }
 
+    public async Task<bool> DeleteEventStream(string liveStreamKey)
+    {
+        if (string.IsNullOrWhiteSpace(liveStreamKey)) throw new ArgumentNullException(nameof(liveStreamKey));
+
+        // endpoint path: use a sensible create path for event streams
+        var request = BuildDelete($"/streams/{liveStreamKey}");
+        var response = await PerformRequest(request);
+        var content = await response.Content.ReadAsStringAsync();
+        Logger.LogInformation("DeleteEventStream response: {Status} {Body}", (int)response.StatusCode, content);
+        try
+        {
+            response.EnsureSuccessStatusCode();
+        }
+        catch (HttpRequestException ex)
+        {
+            Logger.LogError(ex, "Error deleting event stream {LiveStreamKey}. HTTP {StatusCode}. Response body: {Body}", liveStreamKey, (int)response.StatusCode, content);
+            return false;
+        }
```

Comment:
Doesn't need to be a try/catch, same in GetEventStreams


```suggestion
        if (!response.IsSuccessStatusCode)
        {
            Logger.LogError("Error deleting event stream {LiveStreamKey}. HTTP {StatusCode}. Response body: {Body}", liveStreamKey, (int)response.StatusCode, content);
            return false;
        }
```